### PR TITLE
Display fluid send transactions in Qt UI

### PR DESCRIPTION
**Pull request by AmirAbrams**
### What is the purpose of this pull request (PR)?
Fix the outbound fluid transaction display in the QT UI.

Transaction List Before Commit:
dynamic-fluid-tx7-previous

Transaction List After Commit:
dynamic-fluid-tx6

Overview Page Before Commit:
dynamic-fluid-tx-overview_old

Overview Page After Commit:
dynamic-fluid-tx-overview
#### Was this PR tested and how? (If testing is not needed, please explain why)
Yes, using Ubuntu 16.04 on testnet.
#### Does this PR resolve an open issue (reference issue #)?
Yes, resolves issue #32 (Fix fluid transaction display in the QT UI)

### CODE REVIEW

File: src/qt/guiconstants.h [hunk 1]
```diff
@@ -36,7 +36,8 @@ static const bool DEFAULT_SPLASHSCREEN = true;
 #define COLOR_TX_STATUS_DANGER QColor(200, 100, 100)
 /* Transaction list -- TX status decoration - default color */
 #define COLOR_BLACK QColor(0, 0, 0)
-
+/* Transaction list -- Fluid TX status decoration - blue */
```
Comment (Duality-CDOO) [inline src/qt/guiconstants.h]: change "blue" to "default color"

File: src/qt/guiconstants.h [hunk 2]
```diff
@@ -36,7 +36,8 @@ static const bool DEFAULT_SPLASHSCREEN = true;
...
+#define COLOR_FLUID_TX QColor(0, 100, 255)
```
Comment (Duality-CDOO) [inline src/qt/guiconstants.h]: add a comment at the end of the line "// blue"

**Comment by AmirAbrams** [comment]
I made the requested changes.

**Comment by Duality-CDOO** [comment]
This commit looks complete. Are you happy and have nothing else to add? If so, I will merge.

**Pull request merged**